apps/token/src/i18n/translations/dev.json

```diff
@@ -0,0 +1,545 @@
+{
```

Comment:
should have a discussion around how we want to handle this

Reply:
So as discussed. Lets leave in i18nnext but pull our crowdin for now